### PR TITLE
Scene Loading Framework

### PR DESCRIPTION
**Pull request by DanielRihm**
- Added `SceneManager` static class
- Added `SceneEnum` for tracking the individual scenes
- Version is now on top of the other canvases

todo:
- `SceneManager` should probably use a state machine to better track valid transitions
- Some sort of factory or other pattern should be used so the scene names aren't just hard coded into `SceneManager`
- Remove the cube

### CODE REVIEW

**Comment by bitblip** [comment]
How do we support the custom splash screen? A state machine does seem useful here. `InitialScene` -> `CustomSplashScene` -> `TitleScene`. Would it be the responsibility of the SceneManager to delay on the custom splash? I can imagine the CustomSplaceScene should be responsible for this. Something like running a CoRoutine for a few seconds an then call `SceneManager.Next()`?

As a further todo, a useful enhancement would be to load a loading scene while we wait on the target scene `LoadSceneAsync`.

File: Assets/Scripts/Util/SceneEnum.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿namespace LCPS.SlipForge.Enums
```
Comment (DanielRihm) [inline Assets/Scripts/Util/SceneEnum.cs:1]:
```suggestion
﻿namespace LCPS.SlipForge.Enum
```

File: Assets/Scripts/Util/SceneManager.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
+﻿using LCPS.SlipForge.Enums;
```
Comment (DanielRihm) [inline Assets/Scripts/Util/SceneManager.cs:1]:
```suggestion
﻿using LCPS.SlipForge.Enum;
```

**Review by DanielRihm — COMMENTED** [review]
change `LCPS.SlipForge.Enums` to `LCPS.SlipForge.Enum`

**Pull request merged**